### DISCUSSION TITLE
自动阅读会卡住

### DISCUSSION THREAD

**Discussion by heheda123123** [anghunk/linuxdo-scripts#23] [General]
往下滚动的时候，滚一会儿会自己就停住

**Comment by anghunk** [comment]
是不是帖子阅读完了啊？

**Comment by heheda123123** [comment]
没，读几页就卡住了

**Comment by heheda123123** [comment]
比如这个帖子 https://linux.do/t/topic/180489/41
速度50，就会停在这里（是不是太快了，然后下面的帖子还没刷出来就认为到结尾了？）
![image](https://github.com/user-attachments/assets/2c539c9c-62c8-498e-9f57-425bb6182eb0)

**Comment by anghunk** [comment]
@heheda123123 是不是中途离开网页啦，去别的标签页了

暂时不支持离开网页滚动，可以单独拉出来作为一个窗口

**Comment by heheda123123** [comment]
没离开，就在当前页面

**Comment by anghunk** [comment]
> 没离开，就在当前页面

那有可能跟网速有关系，帖子还没刷新出来，然后滚动到底部时自动停了 😂 回头我在弱网环境下测试看看

**Comment by heheda123123** [comment]
谢谢。目前判断页面结尾是怎么判断的？如果遇到结尾了是否可以sleep几秒再判断一次呢

**Comment by anghunk** [comment]
> 谢谢。目前判断页面结尾是怎么判断的？如果遇到结尾了是否可以sleep几秒再判断一次呢

我没手动加判断，是浏览器自己检测到底部了，所以停止不动。
这个 bug 我需要手动优化一下。

**Comment by heheda123123** [comment]
嗯，可能就是她这个“浏览器自己检测到底部了”的逻辑有问题，这种滑到底部再请求刷新，响应慢了就认为到了底部

**Comment by anghunk** [comment]
0.3.4 版本使用代码重构，性能应该会有所提升，可以尝试这个问题是否还存在

**Comment by heheda123123** [comment]
刚刚更新到0.3.4之后试了下，还是有这个问题，是否可以用这个地方的数字作为结尾判断呢
![image](https://github.com/user-attachments/assets/f470c8f2-33a0-49e9-82a6-8860a97c02f1)
